modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/http/http.js

```diff
@@ -220,16 +220,16 @@ exports.Client = function () {
   }
 
   function getUrlParametersFromTupel(tupelParameters) {
-    var queryParameterPairs = [];
-    for (var i = 0; i < tupelParameters.length; i++) {
-      var name = tupelParameters[i].name;
-      var value = tupelParameters[i].value;
+    let queryParameterPairs = [];
```

Comment:
Please, use `const` instead of `let`. Same goes for other variables

Reply:
We push values in queryParametersPairs later in the function. Const will not let us do this.

Reply:
This is not correct. const means the variable can't be reassigned, not that you can't modify it. Refer to MDN for additional docs